src/app/views.py

```diff
@@ -822,7 +822,7 @@ def convert(request):
                     myfile = request.FILES['file']
                     fs = FileSystemStorage(location=settings.MEDIA_ROOT +"/"+ folder,base_url=urljoin(settings.MEDIA_URL, folder+'/'))
                     filename = fs.save(myfile.name, myfile)
-                    uploaded_file_url = fs.url(filename)
+                    uploaded_file_url = fs.url(filename).replace("%20", " ")
```

Comment:
There's one more https://github.com/spdx/spdx-online-tools/blob/f067c36c3822623a257d1d58a5481c9fe71819a6/src/app/views.py#L1156 while uploading xml file

Reply:
I had seen this earlier also but did not change it as it does not gives any error while uploading an XML file. It is mainly due to the reason that we are using the `filename` instead of `uploaded_file_url` in the `xml_upload` function. `uploaded_file_url` is of no use in that function. Should I replace it there also or remove that variable?

Reply:
Sure, go ahead.

Reply:
You can remove the ununsed variables